templates/http-c/content/spin.toml

```diff
@@ -0,0 +1,15 @@
+spin_version = "1"
+authors = ["{{authors}}"]
+description = "{{project-description}}"
+name = "{{project-name}}"
+trigger = { type = "http", base = "{{http-base}}" }
+version = "0.1.0"
+
+[[component]]
+id = "{{project-name | kebab_case}}"
+source = "main.wasm"
+[component.trigger]
+route = "{{http-path}}"
+executor = { type = "wagi" }
+[component.build]
+command = "zig build-exe -O ReleaseSmall -target wasm32-wasi main.c -lc"
```

Comment:
I feel like this might be surprising to someone expecting a C template (even though in practice it's probably a sight more convenient and predictable than WASI SDK Clang).  Perhaps the description could mention e.g. "using C with the Zig toolchain"?

Reply:
Good idea. 💯 agree.